### PR TITLE
fix multiscale discr logits

### PR DESCRIPTION
**Pull request by jpfeil**

### CODE REVIEW

**Comment by lucidrains** [comment]
thanks! haven't added all the multiscale discriminator code just yet, but should be done this week
